server/bundler/loader.js

```diff
@@ -19,7 +19,7 @@ function getSectionsModule( sections ) {
 		].join( '\n' );
 
 		sections.forEach( function( section ) {
-			loadSection += singleEnsure( section.name );
+			loadSection += singleEnsure( section.name, section.module );
```

Comment:
all of this was an attempt to rationalize how webpack sees our dependency tree. It didn't work.